docs/content.zh/docs/connectors/table/hive/overview.md

```diff
@@ -41,7 +41,7 @@ Flink 与 Hive 的集成包含两个层面。
 
 ## 支持的Hive版本
 
-Flink 支持一下的 Hive 版本。
+Flink 支持以下的 Hive 版本。
 
 - 1.0
```

Comment:
I found FLINK-30592.  We have droped the support for hive 1.x, 2.0,2.1,2.2. Could you please also remove it so that we can fix FLINK-30592 by the way in this pr.

Reply:
Thank you for your reply, FLINK-30592 is also submitted by me, I will fix it together in this pr

Reply:
> I found [FLINK-30592](https://issues.apache.org/jira/browse/FLINK-30592). We have droped the support for hive 1.x, 2.0,2.1,2.2. Could you please also remove it so that we can fix [FLINK-30592](https://issues.apache.org/jira/browse/FLINK-30592) by the way in this pr.

Thank you for your reply, [FLINK-30592](https://issues.apache.org/jira/browse/FLINK-30592) is also submitted by me, I will fix it together in this pr